bindings_generator/src/classes.rs

```diff
@@ -23,6 +23,53 @@ pub struct {name} {{
     Ok(())
 }
 
+pub fn generate_class_constants(output: &mut impl Write, class: &GodotClass) -> GeneratorResult {
+    if class.constants.is_empty() {
+        return Ok(());
+    }
+
+    writeln!(output, "/// Constants")?;
+    writeln!(output, "#[allow(non_upper_case_globals)]")?;
+    writeln!(output, "impl {} {{", class.name)?;
+
+    for (name, value) in &class.constants {
+        writeln!(
+            output,
+            "    pub const {name}: i64 = {value};",
+            name = name,
+            value = value,
+        )?;
+    }
+
+    writeln!(output, "}}")?;
+    Ok(())
+}
+
+#[derive(Copy, Clone, PartialEq)]
+struct EnumReference<'a> {
+    class: &'a str,
+    enum_name: &'a str,
+    enum_variant: &'a str,
+}
+
+const ENUM_VARIANTS_TO_SKIP: &[EnumReference<'static>] = &[
```

Comment:
Is it necessary to skip deprecated constants? It might not be very sustainable to manually maintain a list of exceptions, when there are so many classes and constants around. Maybe it's better to just leave it to Godot's documentation?

Reply:
Ah. I see it's required for Rust enum generation. But methods still take `i64`s, so these might not be as useful as the associated constants, that don't require this uniqueness?

Reply:
The associated constants do exist as well, but most of the enums (all except the two that are special cased here) have distinct variants, so for me this is a matter of deprecated items and a single weird "Default" enum that I think should probably not be there in the first place.

That said I agree with you, enums are only a subset of the associated constants and have more limitations, but when these limitations are dealt with properly they are quite nice to use (being able to do an exhaustive match for example).